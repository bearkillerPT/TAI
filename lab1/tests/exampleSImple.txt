ab ac ad af f f f d