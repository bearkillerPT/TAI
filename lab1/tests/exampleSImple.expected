{'a': {'b': 0.3, 'c': 0.3, 'd': 0.3, 'f': 0.3}, 'f': 2.3, 'd': 0.3}